packages/profile-sync-controller/src/constants.ts

```diff
@@ -0,0 +1,14 @@
+const AUTH_ENDPOINT = 'https://authentication.api.cx.metamask.io';
+export const AUTH_NONCE_ENDPOINT = `${AUTH_ENDPOINT}/api/v2/nonce`;
+export const AUTH_LOGIN_ENDPOINT = `${AUTH_ENDPOINT}/api/v2/srp/login`;
+
+const OIDC_ENDPOINT = 'https://oidc.api.cx.metamask.io';
+export const OIDC_TOKENS_ENDPOINT = `${OIDC_ENDPOINT}/oauth2/token`;
+export const OIDC_CLIENT_ID = '1132f10a-b4e5-4390-a5f2-d9c6022db564';
```

Comment:
Is this a secret? Should this be in the repo?

Reply:
Sure, once we get this merged, I can create a additional PR just to handle this change.